google_drive/tests/integration/calendar_base_test.py

```diff
@@ -0,0 +1,17 @@
+""" Google Calendar integration tests """
```

Comment:
Current thought seemed to be that bok choy cannot be ran on XBlock only tests, so we used selenium as suggested instead.